### ISSUE TITLE
Добавить новый язык для подсветки ANTLR

### ISSUE THREAD

**Issue by KvanTTT** [limonte/dear-habr#108]
Все же встречаются правила грамматик на нем.

**Comment by podivilov** [comment]
В данном случае разумнее было бы сразу тогда и BNF/EBNF.

**Comment by daler-als** [comment]
@KvanTTT, @podivilov, для подсветки мы используем https://highlightjs.org, поэтому, если эти языки есть в библиотеке, они могут появиться и на Хабре. 

Недавно так мы добавили поддержку PostgreSQL: https://habr.com/company/postgrespro/blog/430040/.

**Comment by PastorGL** [comment]
Да, хотелось бы. Но https://github.com/highlightjs/highlight.js/pull/1565 — висит уже полтора года без ревью :(

**Comment by daler-als** [comment]
@PastorGL, а попробуйте посоветоваться с Егором Роговым: https://habr.com/ru/users/erogov/. Он недавно успешно добавил в библиотеку поддержку PostgreSQL. Вдруг у него есть хитрости как ускорить процесс?

**Comment by KvanTTT** [comment]
@PastorGL посмотрел я риквест, и изменения мне не очень нравятся. Написал о замечаниях и предложениях там.
